python/cudf/cudf/core/index.py

```diff
@@ -1823,22 +1852,31 @@ def _initialize(self, values, **kwargs):
     def _values(self):
         return next(iter(self._data.columns))
 
-    def copy(self, deep=True):
+    def copy(self, deep=True, name=None, dtype=None):
```

Comment:
Similarly need to update docstring as other instance.

```suggestion
    def copy(self, name=None, deep=False, dtype=None, names=None):
```